Gems/ROS2/Code/CMakeLists.txt

```diff
@@ -66,6 +66,7 @@ ly_add_target(
             Gem::Atom_Component_DebugCamera.Static
```

Comment:
This could be merged with some polishing and renaming.
I suggest putting it in:
Utilities/NavigationTools or similar